shift-react-app/src/steps.ts

```diff
@@ -0,0 +1,65 @@
+// tslint:disable:no-console
+import {
+  readFileSync,
+  writeFileSync,
+} from 'fs';
+import { join } from 'path';
+import findLastIndex from 'lodash.findlastindex';
+import { spawn } from 'child_process';
+
+export function setupProxy() {
+  const proxyFile = join('src', 'setupProxy.js');
+  const proxyCode = `const { setupProxy } = require('@binaris/shift-local-proxy');
+module.exports = setupProxy(__dirname);
+`;
+  writeFileSync(proxyFile, proxyCode, { flag: 'wx' });
```

Comment:
Let's verify the contents have been modified before failing.

Reply:
allow multiple executions of the script if the file content is the same

Reply:
In case it fails for some reason (like bad internet connection) at a later stage.